Firstcomponent.html

```diff
@@ -0,0 +1,61 @@
+<!DOCTYPE html>
+<html>
+<head>
+	<link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Poppins">
+	<link rel="stylesheet" type="text/css" href="first.css">
```

Comment:
Always name CSS files with meaningful name - since we're developing components let's stick to some standards like styles.css and main.js in every components. 

Rename first.css to styles.css